packages/zipkin/index.d.ts

```diff
@@ -368,6 +367,16 @@ declare namespace zipkin {
       recordError(traceId: TraceId, error: Error): void;
     }
   }
+  namespace propagation {
+      interface Propagation {
+        extractor<T>(tracer: Tracer, readHeader: <T> (header: string) => option.IOption<T>): TraceId;
```

Comment:
If I understand correctly, we pass the tracer to the extractor?

Reply:
In new tracing models we have propagation decoupled from tracer, I think we should follow similar pattern here.

Reply:
agree we should not couple these